src/vm/yieldprocessornormalized.cpp

```diff
@@ -31,11 +32,13 @@ void InitializeYieldProcessorNormalized()
     // Intel pre-Skylake processor: measured typically 14-17 cycles per yield
     // Intel post-Skylake processor: measured typically 125-150 cycles per yield
     const int MeasureDurationMs = 10;
-    const int MaxYieldsPerNormalizedYield = 10; // measured typically 8-9 on pre-Skylake
-    const int MinNsPerNormalizedYield = 37; // measured typically 37-46 on post-Skylake
-    const int NsPerOptimialMaxSpinIterationDuration = 272; // approx. 900 cycles, measured 281 on pre-Skylake, 263 on post-Skylake
+    const int NsPerOptimalMaxSpinIterationDuration = 272; // approx. 900 cycles, measured 281 on pre-Skylake, 263 on post-Skylake
     const int NsPerSecond = 1000 * 1000 * 1000;
 
+    // If this constant is changed, the shift value in YieldProcessorWithBackOffNormalized() should be changed as well
+    const int MaxOptimalMaxNormalizedYieldsPerSpinIteration = 10;
```

Comment:
Where did the value of 10 come from?

Reply:
Updated to calculate the value based on other constants instead of hard-coding something close to it and added some comments.